Pipfile

```diff
@@ -0,0 +1,32 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
+typing = "*"
+aiohttp = "==3.5.1"
+aiomysql = "*"
+aiomeasures = "*"
+docopt = "*"
+pycrypto = "*"
+semver = "*"
+"geoip2" = "*"
+msgpack-python = "*"
+trueskill = "*"
+aiocron = "*"
+marisa-trie = "*"
+oauthlib = "*"
+requests-oauthlib = "*"
+sqlalchemy = "*"
```

Comment:
I would pin the package versions. Reason: stable environment for all.

Reply:
That's what the lock file is for :)

Reply:
But if would like to make some stuff and add another library, my pipenv + environment will load another versions of libraries on `pipenv update/lock`.

Reply:
@norraxx only if you specifically do a general `pipenv update`, otherwise the lock-file is respected as-is. You don't want to do that unless you want to  do that.